themes/bootstrap3/theme.config.php

```diff
@@ -4,6 +4,7 @@
     'css' => [
         //'vendor/bootstrap.min.css',
         //'vendor/bootstrap-accessibility.css',
+        'vendor/jquery.dataTables.min.css',
```

Comment:
If this library is only used in one place, would it be better to include it as part of the relevant template(s) instead of making it a global dependency?

Reply:
This seems like a good idea - the templates are moved.